### PR TITLE
feat: suggest views and table settings

### PR DESCRIPTION
**Pull request by Raubzeug**

### CODE REVIEW

**Comment by Raubzeug** [comment]
closes #858

**Review by artemmufazalov — APPROVED** [review]

**Review by artemmufazalov — CHANGES_REQUESTED** [review]
`CREATE VIEW my_view WITH () AS SELECT 1;`

In `WITH` section settings for tables are suggested

File: src/utils/monaco/yql/constants.ts [hunk 1]
```diff
@@ -714,3 +716,36 @@ export const WindowFunctions = [
...
+    view: ['security_invoker'],
+    topic: [
+        'min_active_partitions',
+        'partition_count_limit',
+        'retention_period',
+        'retention_storage_mb',
+        'partition_write_speed_bytes_per_second',
+        'partition_write_burst_bytes',
+        'metering_mode',
+    ],
```
Comment (artemmufazalov) [inline src/utils/monaco/yql/constants.ts:742]: Topic settings are not suggested

File: src/utils/monaco/yql/generateSuggestions.ts [hunk 2]
```diff
@@ -406,6 +413,20 @@ export async function generatePragmasSuggestion(
         sortText: suggestionIndexToWeight(getSuggestionIndex('suggestPragmas')),
     }));
 }
+export async function generateEntitySettingsSuggestion(
+    rangeToInsertSuggestion: monaco.IRange,
+    entityType: YQLEntity,
+): Promise<monaco.languages.CompletionItem[]> {
+    const tableHints = await getEntitySettings(entityType);
```
Comment (artemmufazalov) [inline src/utils/monaco/yql/generateSuggestions.ts:420]: Please, rename `tableHints` to `entityHints`

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**